lib/server.js

```diff
@@ -675,12 +675,18 @@ class WebServer {
             //  alarm_arm_away, alarm_arm_home, alarm_arm_night, alarm_arm_custom_bypass, alarm_disarm (code will be sent)
             this.log.debug(data.service + ': ' + id + ' = ' + data.service_data.code ? 'XXX' : 'none');
             this.adapter.getForeignObject(id, (err, obj) => {
-                if (obj.native.alarm_code && obj.native.alarm_code.toString() !== data.service_data.code.toString()) {
+                this.log.debug("native" + JSON.stringify(obj.native));
```

Comment:
Please remove this log output. If there is a code set, this would print it to the log.